test/artefacts/simplified-expenses-checker/car/live_on_business_premises/no/no/new/2345.0/no/250001.0/35.0/10000/789.0/0.txt

```diff
@@ -0,0 +1,40 @@
+
+
+Based on your answers, the expenses you can claim and deduct are listed below. There are 2 sets of results comparing simplified expenses and calculating expenses by working out the actual costs.
+
+##Simplified expenses
+
+- £4,500 to claim for cars and vans
+
+**Total to claim per year:** £4,500
+
+###What to do for your tax return
+
+Use the simplified expenses flat rates to work out your business expenses.
+
+Multiply the flat rates with the [miles you drove](/simpler-income-tax-simplified-expenses/vehicles-), [hours you work from home](/simpler-income-tax-simplified-expenses/business-use-of-your-home) or the [number of people who live on your business premises](/simpler-income-tax-simplified-expenses/private-use-of-business-premises).
+
+[Self Assessment tax returns](/self-assessment-tax-returns)
+
+##Calculating expenses by working out the actual costs
+
+- £2,345 for cars and vans and motorcycles
+
+- £15,750.06 for for cars, vans and motorcycles
```

Comment:
https://smart-answers-pr-2900.herokuapp.com/simplified-expenses-checker/y/car/live_on_business_premises/no/no/new/2345.0/high/250001.0/35.0/10000/789.0/0

@leenagupte This is a very huge observation and thanks for spotting this out.. it never crossed my mind.

From this duplication from 5e5705d0c064d9b46184e93f0c45c917bd86d1e3, it appears that the vehicle emission question option still used yes/no. From 7d207bd7aa58a8ba5168d4f50cd9ec62d23ca218, new set of options (i.e low/medium/high) were added. See the following artefacts:
```
/car/live_on_business_premises/no/no/new/2345.0/no/250001.0/35.0/10000/789.0/0.txt
/car/live_on_business_premises/no/no/new/2345.0/low/250001.0/35.0/10000/789.0/0.txt
/car/live_on_business_premises/no/no/new/2345.0/medium/250001.0/35.0/10000/789.0/0.txt
/car/live_on_business_premises/no/no/new/2345.0/high/250001.0/35.0/10000/789.0/0.txt
```